gulpfile.js

```diff
@@ -33,17 +39,27 @@ gulp.task('js:test', done => {
 });
 
 gulp.task('js:lint', () => {
-    return gulp.src(lintPaths)
+    return gulp.src(lintPathsJS)
         .pipe(eslint())
         .pipe(eslint.format())
         .pipe(eslint.failAfterError());
 });
 
+
+gulp.task('css:lint', function lintCssTask() {
+    return gulp.src(lintPathsCSS)
+        .pipe(gulpStylelint({
+            reporters: [
+                {formatter: 'string', console: true}
+            ]
+        }));
+});
+
 gulp.task('static:clean', () => {
     return del(['static/**', '!static', '!static/.gitignore']);
 });
 
 gulp.task('default', () => {
     gulp.start('media:watch');
-    gulp.watch(lintPaths, ['js:lint']);
+    gulp.watch(lintPathsJS, ['js:lint']);
```

Comment:
I changed our default gulp task to lint both CSS and JS files, but for changed files only instead of the whole folders. This speeds things up considerably 👍